test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -124,9 +121,9 @@ async fn generate_predicate_inputs(amount: u64, wallet: &Wallet) -> (Vec<u8>, Sd
 
     let predicate_root = predicate.address();
 
-    let balance: u64 = wallet.get_asset_balance(&AssetId::default()).await.unwrap();
+    let balance: u128 = wallet.get_asset_balance(&AssetId::default()).await.unwrap();
 
-    assert!(balance >= amount);
+    assert!(balance >= amount as u128);
 
     wallet
```

Comment:
interesting; i would have expect `wallet.transfer` to use `u128` for amount; - probably need to update this in future.
All interfaces using `amount` need to use `u128` i'd imagine; otherwise we'd need to do these intermediary conversions in code :/